services/database/programcardinfo.ts

```diff
@@ -0,0 +1,168 @@
+import prisma from "@database";
```

Comment:
Let's try to keep naming consistent. I think `program-card-info.ts` would be better? What do you guys think? @mahadzaryab1 @cindy-wang328

Reply:
![image](https://user-images.githubusercontent.com/50597354/128642183-69963a18-f062-4fd7-832d-c41b94223d87.png)
sure but LOL i was following these

Reply:
I talked to @mahadzaryab1 and he said we should follow camelCase for typescript conventions actually